css/styles.css

```diff
@@ -0,0 +1,98 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Crimson+Pro:wght@200..900&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  text-align: center;
+  align-items: center;
+  background-color: #1c1a19;
+}
+
+/* Flex */
+.nav,
+.social-media {
+  display: flex;
+  order: 0;
+  justify-content: space-between;
+  background: #3c3a39;
+  padding: 0 2rem;
+}
```

Comment:
- [OPTIONAL] I strongly recommend you to give it a margin top and bottom its good for your design and also please don't forget to change your humbergur bottom color into white.
### your design👇
![padding](https://user-images.githubusercontent.com/66992901/198520647-3331aee7-1604-43fb-ab1b-44ca718a9775.PNG)

Reply:
Thanks again! Will do.